pinecone/index_connection.go

```diff
@@ -996,8 +1003,307 @@ func (idx *IndexConnection) DescribeIndexStatsFiltered(ctx context.Context, meta
 	}, nil
 }
 
-func (idx *IndexConnection) query(ctx context.Context, req *data.QueryRequest) (*QueryVectorsResponse, error) {
-	res, err := (*idx.dataClient).Query(idx.akCtx(ctx), req)
+// StartImportResponse holds the response parameters for the StartImport method.
+//
+// Fields:
+//   - Id: The ID of the import process that was started.
+type StartImportResponse struct {
+	Id string `json:"id,omitempty"`
+}
+
+// StartImport imports data from a storage provider into an index. The uri parameter must start with the
+// schema of a supported storage provider. For buckets that are not publicly readable, you will also need to
+// separately configure a storage integration and pass the integration id.
+//
+// Returns a pointer to a StartImportResponse object with the import ID or an error if the request fails.
+//
+// Parameters:
+//   - ctx: A context.Context object controls the request's lifetime,
+//     allowing for the request to be canceled or to timeout according to the context's deadline.
+//   - uri: The URI of the data to import. The URI must start with the scheme of a supported storage provider.
```

Comment:
`scheme` >> `schema`

Reply:
I think it's supposed to be scheme, right? https://en.wikipedia.org/wiki/List_of_URI_schemes

Reply:
oh idk, you just say `schema` above, so I thought it should be `schema` :)